File: Cpp/String/RotateString.cpp
//https://leetcode.com/problems/rotate-string/description/
class Solution {
public:
    bool rotateString(string s, string goal) {
        for(int i=1;i<=s.length();i++){
        string var=s.substr(i,s.length()-i)+s.substr(0,i);
    
    if(var.compare(goal)==0){
      return true;
    }
   }
        return false;
    }
};

File: Python/String/Rotate String.py
//https://leetcode.com/problems/rotate-string/
class Solution:
    def rotateString(self, s: str, goal: str) -> bool:
        for i in range(1, len(s) + 1):
            var = s[i:] + s[:i]
            if var == goal:
                return True
        return False
